platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/stats/ReconnectMapMetrics.java

```diff
@@ -29,6 +29,10 @@ public class ReconnectMapMetrics implements ReconnectMapStats {
 
     private static final String RECONNECT_MAP_CATEGORY = "reconnect_vmap";
 
+    /** A map label as passed to the constructor, w/o any normalization. */
```

Comment:
curious - what does `normalization` mean here? If you have any round of changes to add, do you mind explaining it? Otherwise it's a super minor thing so it can be added in a later PR. Changes look good to me

Reply:
This refers to https://github.com/hashgraph/hedera-services/blob/245a322d4b856075d96fee6957045707dec262a3/platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/stats/ReconnectMapMetrics.java#L66-L67

just below in this same class.